tests/unit/rules/javascript/test.deprecated_browser_apis.js

```diff
@@ -0,0 +1,19 @@
+import { VALIDATION_WARNING, DEPRECATED_JAVASCRIPT_APIS } from 'const';
+import JavaScriptScanner from 'scanners/javascript';
+
+import { validMetadata } from '../../helpers';
+
+describe('deprecated browser APIs', () => {
+  DEPRECATED_JAVASCRIPT_APIS.forEach((api) => {
+    it(`should return deprecation warning when ${api} is used`, async () => {
+      const fakeMetadata = { addonMetadata: validMetadata({}) };
+      const code = `chrome.${api}();`;
```

Comment:
Nit, we may also include a call that uses `browser.${api}();` and explicitly check that we get 2 warning messages for that API.